src/common/actions/snaps.js

```diff
@@ -9,16 +9,15 @@ export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
 export const FETCH_SNAPS_SUCCESS = 'FETCH_SNAPS_SUCCESS';
 export const FETCH_SNAPS_ERROR = 'FETCH_SNAPS_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps() {
```

Comment:
I'm not sure yet; that's why I'd propose leaving this be until we have a chance to work that out, rather than having code churn that may turn out to be unnecessary.